sklearn/metrics/cluster/supervised.py

```diff
@@ -193,9 +195,9 @@ def adjusted_rand_score(labels_true, labels_pred, max_n_classes=5000):
     # Special limit cases: no clustering since the data is not split;
     # or trivial clustering where each document is assigned a unique cluster.
     # These are perfect matches hence return 1.0.
-    if (classes.shape[0] == clusters.shape[0] == 1
-            or classes.shape[0] == clusters.shape[0] == 0
-            or classes.shape[0] == clusters.shape[0] == len(labels_true)):
+    if (classes.shape[0] == clusters.shape[0] == 1 or
```

Comment:
Sorry, you're right.
I will do the pep8 corrections with a different PR next time.